models/classes/class.CrudQtiTestsService.php

```diff
@@ -91,8 +92,15 @@ public function importQtiTest(
                 $importer->enableItemMustBeOverwritten();
             }
 
-            $report = $importer->importMultipleTests($class, $uploadedFile, $overwriteTest, $itemClassUri);
+            $report = $importer->importMultipleTests(
+                $class,
+                $uploadedFile,
+                $overwriteTest,
+                $itemClassUri,
+                $overwriteTestUri
+            );
             helpers_TimeOutHelper::reset();
+
             return $report;
         } catch (common_exception_UserReadableException $e) {
             return new common_report_Report(common_report_Report::TYPE_ERROR, __($e->getUserMessage()));
```

Comment:
@shpran Can you please add a proper log here like explained here?

https://github.com/oat-sa/extension-tao-testqti/pull/2491/files#r1703668338

Thank you!

Reply:
https://github.com/oat-sa/extension-tao-testqti/pull/2491/commits/f6c647fbb273b52b4e0e2b72399a6812444856da